lib/ferrum/page.rb

```diff
@@ -108,6 +108,16 @@ def resize(width: nil, height: nil, fullscreen: false)
                                                     fitWindow: false)
     end
 
+    def get_position
+      @window_id = @browser.command("Browser.getWindowForTarget", targetId: @target_id)["windowId"]
+      @browser.command("Browser.getWindowBounds", windowId: @window_id).fetch("bounds").values_at("left", "top")
+    end
+
+    def set_position(left, top)
+      @window_id = @browser.command("Browser.getWindowForTarget", targetId: @target_id)["windowId"]
+      @browser.command("Browser.setWindowBounds", windowId: @window_id, bounds: { left: left, top: top })
+    end
```

Comment:
```suggestion
    def get_position
      @browser.command("Browser.getWindowBounds", windowId: window_id).fetch("bounds").values_at("left", "top")
    end

    def set_position(left, top)
      @browser.command("Browser.setWindowBounds", windowId: window_id, bounds: { left: left, top: top })
    end
```

Reply:
now here we can use `window_id` method:
https://github.com/rubycdp/ferrum/pull/116/files#diff-2f9537c613f48bed5ff6ca273e2dac28886a32fe78dcd2dc7c7c798c891434d3L140-L142